### PR TITLE
Added Bulgarian and Swedish as new languages

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
We have 2 new translated languages for Mattermost:
Bulgarian and Swedish. There are added to the documentation.

### CODE REVIEW

File: source/overview/product.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Features include:
 - Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`__ and `third party extensions <https://about.mattermost.com/default-app-directory/>`__
 - Easily scalable to dozens of users per team
 - New features and improvements released every two months
-- Multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian
+- Multiple languages including U.S. English, Chinese (Simplified and Traditional), Bulgarian, Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, Swedish and Ukrainian
```
Comment (justinegeffen) [inline source/overview/product.rst:43]:
```suggestion
- Multiple languages including U.S. English, Chinese (Simplified and Traditional), Bulgarian, Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, Swedish, and Ukrainian
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks so much, @ctlaltdieliet! One minor punctuation edit on my side but otherwise LGTM!

**Review by jasonblais — APPROVED** [review]
Thank you!

**Review by jasonblais — CHANGES_REQUESTED** [review]
@ctlaltdieliet Here, we can have Bulgarian before Chinese (we list them in alphabetical order in the product overview, after U.S. English).

**Comment by amyblais** [comment]
Completed via https://github.com/mattermost/docs/pull/4274

**Pull request closed without merging**